1_clusters/binary-auth.tf

```diff
@@ -30,6 +30,30 @@ resource "google_kms_key_ring" "keyring" {
   depends_on = [null_resource.resource-to-wait-on]
 }
 
+# Create a Google Secret containing the keyring name
```

Comment:
Does this really need to be stored as a secret? I'm a little unsure on the intent here.

Reply:
Got it, thanks for explaining.

Reply:
Is this a LGTM statement? Just need someone to verily things look good

Reply:
Yeah, LGTM.